docs/source/modules/mixedlayer.rst

```diff
@@ -0,0 +1,277 @@
+Mixed layer module
+=====================
+
+Summary
+----------------------
+This module updates the sea surface temperature (SST, :math:`T_s`). 
+
+SST boundary condition options
+-----------------------
+The SST options are:
+    - an input SST file (``do_sc_sst`` namelist option)
+    - a slab ocean aquaplanet analytic form (``prescribe_initial_dist`` namelist option)
+    - an AquaPlanet Experiment protocol (APE) analytic form (``do_ape_sst`` namelist option)
+    - an implicit timestep (``do_calc_eff_heat_cap`` namelist option)
+    - an optional Q-flux (this is not controlled by a flag but rather an input file that is generated offline).
+
+Each of these options are discussed below.
+
+Input SST file
+^^^^^^^^^^^^^^^
+Set ``do_sc_sst`` to True if you want to specify the SST field. The SST field will be read in from an input filename using the ``sst_file`` variable. 
+Using an input SST field is useful, for example,  when you want to add a temperature anomaly. The file is read in during the initialisation (i.e. within the call to mixed_layer_init from within idealized_moist_phys).
+The input file can be time independent (i.e. no diurnal or seasonal cycle or any changes in the SST from one time step to another) or vary with time. More information can be found in the Diagnostics section below.
+
+Slab ocean 
+^^^^^^^^^^^^^^^
+During the initialisation, if there is no restart file to open, the surface temperature is set to the prescribed initial distribution (``prescribe_initial_dist = True``)
+
+:math:`T_s = T_{surf} -\frac{1}{3}*dT*\left(3\sin(\lambda)^2-1\right)`,
+
+where the default values we use in the trip test for the Frierson test case (exp/test_cases/frierson/frierson_test_case.py) are: :math:`T_{surf} = 285 K` and :math:`dT = 40 K`.
+
+This form of :math:`T_{surf}` is similar to a 2nd legendre polynomial, it is a parabola that maximises at the equator.
+
+APE aquaplanet (analytic form)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+The prescribed SST for the APE aquaplanet protocol is given by
+
+:math:`T_s = 27 \left( 1 - \sin^2\left( \frac{3}{2} \lambda \right) \right)`,
+
+between 60N-60S, equation 1 of Neele and Hoskins 2004 [NealeHoskins2004]_, and 0 deg C poleward of 60N/S.
+
+
+Implicit timestep
```

Comment:
Add you add a general english sentence her eon what implicit time stepping is please.